src/piggui.rs

```diff
@@ -407,9 +438,13 @@ impl Application for Gpio {
                 border_radius: 2.0,
             };
 
-            let version_text = Text::new(version().lines().next().unwrap_or_default().to_string());
-
-            let version_row = Row::new().push(version_text).align_items(Alignment::Start);
+            let about_button_style = CustomButton {
```

Comment:
You can create a toast field in Gpio, and then implement a view() method in Toast struct,
then the Gpio::view() method just calls the Toast::view() method, allowing you to move all the code to the toast.rs file

I'm trying to do that with waveform.rs ....